Milter/utils.py

```diff
@@ -72,7 +72,7 @@ def iniplist(ipaddr,iplist):
   True
   >>> iniplist('4.2.2.2',['b.resolvers.Level3.net'])
   True
-  >>> iniplist('2607:f8b0:4004:801::',['google.com/40'])
+  >>> iniplist('2606:2800:220:1:248:1893:25c8:1946',['example.com/64'])
```

Comment:
The IP for example.com is stable?  Fine, but I need to change the test case to a /64 and /40, as that was the original failure.